emdcbackend/emdcbackend/views/judge.py

```diff
@@ -86,17 +86,91 @@ def create_judge(request):
+                delete_cluster_judge_mapping_by_id_nonhttp(cluster.id)
+                map_cluster_to_judge({
+                    "judgeid": judge.id,
+                    "clusterid": new_cluster
+                })
+
+                # update the boolean values
+                if judge.presentation != new_presentation:
+                    judge.presentation = new_presentation
+                if judge.mdo != new_mdo:
+                    judge.mdo = new_mdo
+                if judge.journal != new_journal:
+                    judge.journal = new_journal
+                if judge.penalties != new_penalties:
+                    judge.penalties = new_penalties
+
+                clusterid = new_cluster
+
+            else:
+                # if adding or removing scoresheets (no cluster change)
+                if new_presentation != judge.presentation and new_presentation == False:
+                        delete_sheets_for_teams_in_cluster(judge.id, clusterid, False, True, False, False)
+                        judge.presentation = False
+                elif new_presentation != judge.presentation and new_presentation == True:
+                        create_sheets_for_teams_in_cluster(judge.id, clusterid, False, True, False, False)
+                        judge.presentation = True
+
+                if new_mdo != judge.mdo and new_mdo == False:
+                        delete_sheets_for_teams_in_cluster(judge.id, clusterid, False, False, False, True)
+                        judge.mdo = False
+                elif new_mdo != judge.mdo and new_mdo == True:
+                        create_sheets_for_teams_in_cluster(judge.id, clusterid, False, False, False, True)
+                        judge.mdo = True
+
+                if new_journal != judge.journal and new_journal == False:
+                        delete_sheets_for_teams_in_cluster(judge.id, clusterid, False, False, True, False)
+                        judge.journal = False
+                elif new_journal != judge.journal and new_journal == True:
+                        create_sheets_for_teams_in_cluster(judge.id, clusterid, False, False, True, False)
+                        judge.journal = True
+
+                if new_penalties != judge.penalties and new_penalties == False:
+                        delete_sheets_for_teams_in_cluster(judge.id, clusterid, True, False, False, False)
+                        judge.penalties = False
+                elif new_penalties != judge.penalties and new_penalties == True:
+                        create_sheets_for_teams_in_cluster(judge.id, clusterid, True, False, False, False)
+                        judge.penalties = True
+
+            judge.save()
+
+        serializer = JudgeSerializer(instance=judge)
+    
+    except Exception as e:
+        raise ValidationError({"detail": str(e)})
+    
+    return Response({"judge": serializer.data, "clusterid": clusterid}, status=status.HTTP_200_OK)
```

Comment:
Format responses to be similar to create_judge, unless frontend has specifically said they don't need that response data.